stdeb/pypi_simple.py

```diff
@@ -0,0 +1,46 @@
+import re
+import requests
+import stdeb
+
+USER_AGENT = "pypi-install/%s ( https://github.com/astraw/stdeb )" % \
+    stdeb.__version__
+
+
+def normalize_package_name(package_name):
+    return re.sub(r"[-_.]+", "-", package_name).lower()
+
+
+class PyPIClient(object):
+    def __init__(self, pypi_url="https://pypi.org", user_agent=USER_AGENT):
+        self.pypi_url = pypi_url
+        self._http_session = requests.Session()
+        self._http_session.headers["User-Agent"] = user_agent
+
+    def release_versions(self, package_name, current_only=False):
+        package_name = normalize_package_name(package_name)
+        if current_only:
```

Comment:
The two code paths determined by `current_only` share only the package name normalization which is already in a separate function. I'd recommend introducing a new function for the `True` path here, especially since the return type seems to change based on that, which is pretty confusing.

Reply:
Great point! I was culting forward a pretty straight port of the interface from the old XML-RPC transport, but even that was a carry-over from an interface that was not actually being used for what it is for. I've split the functions in 2d0a0898a57dc4fab13c63b41216fbf4582257a9 and the result is much better. Thanks for spotting it.